other/materials_designer/specific_examples/interface_bilayer_twisted_commensurate_lattices_molybdenum_disulfide.ipynb

```
@@ -37,16 +48,34 @@
-    "    # {\"angle\": 47.0, \"distance\": 6.5},\n",
-    "    # {\"angle\": 60.0, \"distance\": 6.2},\n",
-    "]\n",
-    "\n",
+    "    {\"name\": \"MoS2 bilayer 21.8deg d6.5\", \"angle\": 21.8, \"d_mo_mo\": 6.5},\n",
+    "    {\"name\": \"MoS2 bilayer AB1 d6.1\", \"angle\": 60.0, \"d_mo_mo\": 6.1},\n",
+    "    {\"name\": \"MoS2 bilayer AB1 d6.5\", \"angle\": 60.0, \"d_mo_mo\": 6.5},\n",
+    "    # {\"name\": \"MoS2 bilayer AA3 d6.8\", \"angle\": 0.0, \"d_mo_mo\": 6.8},\n",
+    "    # {\"name\": \"MoS2 bilayer 13.2deg d6.5\", \"angle\": 13.2, \"d_mo_mo\": 6.5},\n",
+    "    # {\"name\": \"MoS2 bilayer 38.2deg d6.5\", \"angle\": 38.2, \"d_mo_mo\": 6.5},\n",
+    "    # {\"name\": \"MoS2 bilayer 46.8deg d6.5\", \"angle\": 46.8, \"d_mo_mo\": 6.5},\n",
+    "]"
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)